formtron-schema.json

```diff
@@ -52,44 +50,6 @@
         "custom": {}
       },
       "additionalProperties": false
-    },
-    "array": {
-      "type": "object",
-      "required": ["title", "type", "items"],
-      "properties": {
-        "type": {
-          "type": "string",
-          "enum": ["array"]
-        },
-        "title": {
-          "type": "string"
-        },
-        "path": {
-          "type": "string"
-        },
-        "items": { "$ref": "#/definitions/field" },
-        "custom": {}
-      },
-      "additionalProperties": false
-    },
-    "object": {
```

Comment:
Note: this is technically a breaking change, so Formtron 2 should be released.

Reply:
Actually, I don't think _anything_ uses this file. Originally I wanted `ui-schema` to inherit from `formtron-schema` as its starting point but I didn't figure out how to do that.

Reply:
but you're right, we should drop this as 2.0 simply because formerly usable schema types like `"string{}"` will no longer work.